src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -62,14 +70,23 @@ private void processMessage(final ChatMessage chatMessage) {
     public Set<CommandHandle> getCommands() {
         return Collections.unmodifiableSet(commands);
     }
+    
+    public Set<CommandHandle> getListeners(){
+    	return Collections.unmodifiableSet(listeners);
+    }
 
     @Override
     public void subscribe(CommandHandle subscriber) {
-        commands.add(subscriber);
+    	if (subscriber.getName() == null){
+    		listeners.add(subscriber);
+    	} else {
+    		commands.add(subscriber);
+    	}
```

Comment:
This one smells... See above comments about extracting a separate bot. It shouldn't be that much of a problem to set OakBot up as separate class